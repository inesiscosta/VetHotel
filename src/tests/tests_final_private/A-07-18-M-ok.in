8
5
h2
a1
a1
1
3
DECIDUOUS
5
h1
a2
a2
10
4
EVERGREEN
1
0
4
4
3
ap04.dat
0
