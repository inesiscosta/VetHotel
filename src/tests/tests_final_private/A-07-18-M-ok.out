Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|a1|a1|1|3|DECIDUOUS|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|a2|a2|10|4|EVERGREEN|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: HABITAT|h1|nome2|12|1
ÁRVORE|a2|a2|10|4|EVERGREEN|GERARFOLHAS
HABITAT|h2|outro nome|122|1
ÁRVORE|a1|a1|1|3|DECIDUOUS|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: 1
Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: 2
Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: Unnamed File. Save File As: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: 